src/Core/PConfig.php

```diff
@@ -35,12 +35,12 @@ public static function init(Config\PConfiguration $config)
 	/**
 	 * @brief Loads all configuration values of a user's config family into a cached storage.
 	 *
-	 * @param string $uid The user_id
+	 * @param int    $uid The user_id
 	 * @param string $cat The category of the configuration value
 	 *
 	 * @return void
 	 */
-	public static function load($uid, $cat)
+	public static function load(int $uid, $cat)
```

Comment:
Since the typehint for `$cat` is set for the follow-up call, we can add it here as well